### PR TITLE
[PHX-2023] Add 2023 event details

### PR DESCRIPTION
**Pull request by desktophero**
Setting up a draft while we schedule the initial video call with organizers.

### CODE REVIEW

File: data/events/2023-phoenix.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+team_members: # Name is the only required field for team members.
+  - name: "Laurie Biviano"
+  - name: "Blaine Daiutolo"
+  - name: "Ian McClarty"
+  - name: "Justin James"
+  - name: "Jason Walker"
+    twitter: "TheDesktophero"
+    github: "desktophero"
+
+organizer_email: "organizers-phoenix-2017@devopsdays.org" # Put your organizer email address here
```
Comment (yvovandoorn) [inline data/events/2023-phoenix.yml:61]: you can change this to just cityname @ devopsdays.org

Comment (mattstratton) [reply data/events/2023-phoenix.yml:61]: Not only *can* you, but you *should*

Comment (desktophero) [reply data/events/2023-phoenix.yml]: Gotcha - I was following information from the previous PHX events. Updated.

**Comment by desktophero** [comment]
Not sure what happened with all the re-review requests above. 🤷

**Comment by yvovandoorn** [comment]
In order for this PR to be merged a kick off call should take place with the core team. I believe @mattstratton reached out but dates were never confirmed. 

I will be closing this PR in the next few weeks if there is no progress in organizing this call.

**Comment by yvovandoorn** [comment]
Closing until PHX can get a call organized with core.

**Comment by mattstratton** [comment]
Kick off call occurred with Matty on 14 Dec, so re-opening the PR.

@desktophero will need to confirm the dates and ensure the contact page is matching the email alias before this can be merged.

**Pull request closed without merging**